doc/source/ray-core/handling-dependencies.rst

```diff
@@ -601,8 +601,8 @@ the error message in detail.
     a = A.options(runtime_env=bad_env).remote()
     ray.get(a.f.remote())
 
-You can also enable runtime_env debugging logs by setting an environment variable ``RAY_RUNTIME_ENV_LOG_TO_DRIVER_ENABLED=1``. 
-It will print the full runtime_env setup log messages to the driver.
-The same information can be found from the log file ``runtime_env*.log`` from the log directory. 
+You can also enable ``runtime_env`` debugging log streaming by setting an environment variable ``RAY_RUNTIME_ENV_LOG_TO_DRIVER_ENABLED=1`` before starting Ray on the head node (e.g. with ``ray start --head``). 
```

Comment:
@rkooo567 let me know if this is correct -- I wasn't 100% sure if it needs to be set on all the nodes, or just the head node.

Reply:
It should be on all nodes. In a worker, it should also be set